packages/java/tests/spring/react-grid-test/src/main/resources/data.sql

```diff
@@ -1,53 +1,53 @@
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (30, 0, 'Dylan', 'Fisher',1,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (31, 0, 'Elaine', 'Garcia',123,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (32, 0, 'Frank', 'Hernandez',12,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (33, 0, 'Grace', 'Ingram',11,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (34, 0, 'Henry', 'Jackson',16,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (35, 0, 'Isabella', 'King',10,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (36, 0, 'Jason', 'Lopez',554,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (37, 0, 'Kaylee', 'Miller',3,false);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (38, 0, 'Leo', 'Nelson',13,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (39, 0, 'Morgan', 'Owens',7,false);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (40, 0, 'Nathan', 'Perez',53,false);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (41, 0, 'Oliver', 'Quinn',37,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (42, 0, 'Patricia', 'Roberts',17,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (43, 0, 'Quincy', 'Smith',123,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (44, 0, 'Rebecca', 'Taylor',123,false);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (45, 0, 'Steven', 'Underwood',123,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (46, 0, 'Thomas', 'Vance',25,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (47, 0, 'Uma', 'Washington',14,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (48, 0, 'Vivian', 'Xavier',19,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (49, 0, 'William', 'Young',22,true);
-INSERT INTO person (id, version, first_name, last_name, lucky_number,email_verified) VALUES (50, 0, 'Xander', 'Zane',18,true);
+INSERT INTO person (id, version, first_name, last_name, lucky_number, email_verified, birth_date, shift_start, appointment_time) VALUES (1, 0, 'Alice', 'Johnson', 123, true, '1990-05-15', '09:30', '2023-09-27 08:53:17');
```

Comment:
Quite a busy day that 2023-09-27

Reply:
My prompt skills weren't strong enough to convince GPT to randomize the date. Does it matter?

Reply:
Nope